src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.task.Task;
+
+/**
+ * Encapsulates a Storage
+ */
+public class Storage {
+
+    private static final String FILE_NAME = "data.txt";
+
+    /**
+     * Returns an array list of tasks stored from previous runs
+     *
+     * @return List of Tasks
+     * */
+    public static ArrayList<Task> loadData() {
+        try {
+            FileInputStream fin = new FileInputStream(FILE_NAME);
+            ObjectInputStream ois = new ObjectInputStream(fin);
+            @SuppressWarnings("unchecked") ArrayList<Task> tasks = (ArrayList<Task>) ois.readObject();
```

Comment:
It'd be great if you could also close the FileInputStream after you're done with it (a 'try with resources' statement would be very convenient for your usecase!)